configuration/scripts/ice_in

```diff
@@ -125,12 +126,14 @@
     ktherm            = 2
     conduct           = 'bubbly'
     ksno              = 0.3d0
+    hi_min            = 0.01d0
     a_rapid_mode      =  0.5e-3
     Rac_rapid_mode    =    10.0
     aspect_rapid_mode =     1.0
     dSdt_slow_mode    = -5.0e-8
     phi_c_slow_mode   =    0.05
     phi_i_mushy       =    0.85
+    Tliquidus_max     = -0.1d0
```

Comment:
Consider Tliquidus_max = 0 as in cicecore/cicedyn/general/ice_init.F90 and doc/source/user_guide/ug_case_settings.rst so they all match

Reply:
This is in the standalone test scripts, unrelated to our use as a coupled component.